src/challenges.js

```diff
@@ -28,18 +36,22 @@ const cacheFunction = cb => {
 
 /* ======================== Recursion Practice ============================ */
 const reverseStr = str => {
-  // reverse str takes in a string and returns that string in reversed order
-  // The only difference between the way you've solved this before and now is that you need to do it recursivley!
+  var first = str[0];
```

Comment:
I would use `const` until you're not able to. Then use `let`. There is much more power in understanding const and let and their proper use cases these days. Using `var` tells me you want to be able to redefine the variable in the same scope later on. This idea of `overriding` variables is now solved with the new keywords.